### ISSUE TITLE
Feedback and code help 

### ISSUE THREAD

**Issue by tatyanaberdan** [alliefoard/hello-world#5]
I really liked the colors and design of your site, especially the box you put around your header. If you are going to be making changes, I would recommend putting your social media buttons higher up - maybe in a sidebar? - and making them bigger and more prominent.  

Also, about this code in your CSS:

/* NEED TO FIGURE OUT*/ NEED HELP *****
 h1.entry-title {
 	text-align: center;
 }

have you tried to just style ".entry-title," rather than "h1.entry-title"?

**Comment by alliefoard** [comment]
Yes, I have and still haven't figured it out! Working on it, thanks!
